### PR TITLE
Colorpicker: swatch: show swatch for transparent colors; fixed color …

### LINKED ISSUE #1995: css transparent on click doesn't open color picker
### Short Summary
When you click on the color value transparent, you will get an not implemented error.

### Steps to Reproduce
Click on the color transparent value ;)

### Expected results
Color picker should open with full opacity 

### Actual results
Not implemented error is triggerd

### Platform Information
Ide X 10.1 - windows 10

### PR DESCRIPTION
**Pull request by Defman21**
…picker not working with rgba with alpha = 0; fixed #1995

Signed-off-by: Defman21 <i@defman.me>

### CODE REVIEW

File: src/chrome/komodo/content/hyperlinks/csscolorpicker.js [hunk 1]
```diff
@@ -387,7 +388,13 @@ ko.hyperlinks.ColorPickerHandler.prototype.colorToHex = function(color) {
 ko.hyperlinks.ColorPickerHandler.prototype.colorToRGB = function(color) {
     var span = document.createElement('span');
     span.style.color = color;
-    return window.getComputedStyle(span, null).color;
+    var _color = window.getComputedStyle(span, null).color;
```
Comment (Naatan) [inline src/chrome/komodo/content/hyperlinks/csscolorpicker.js]: Nitpick time! This need to be named better, the variable name should reflect what it's holding.

Comment (Defman21) [inline src/chrome/komodo/content/hyperlinks/csscolorpicker.js]: Do'h :P

**Comment by Naatan** [comment]
Thanks @Defman21 ! :)

**Comment by Naatan** [comment]
Bah, just noticed something after I merged - you are returning a value but this function was never returning anything. It was setting the value of the input. Can you fix that please?

**Comment by Defman21** [comment]
The function was returning a value.
https://github.com/Komodo/KomodoEdit/pull/2075/files#diff-ad2e92594d1e025ecb6f699f7ba03a83L390

**Comment by Naatan** [comment]
Sigh, github diff is really broken apparently, cause that wasnt evident here:

https://github.com/Komodo/KomodoEdit/commit/ba79537d39e4c24e0f6bd42fb6a0ca44809496c2

Thanks @Defman21

**Pull request merged**